src/libsystemd-network/ndisc-neighbor-internal.h

```diff
@@ -7,10 +7,10 @@
 #include "set.h"
 
 struct sd_ndisc_neighbor {
-        unsigned n_ref;
-
         ICMP6Packet *packet;
 
+        unsigned n_ref;
```

Comment:
as mentioned, please don't move n_ref around

Reply:
> as mentioned, please don't move n_ref around

sorry, I didn't notice about n_ref, maybe I should move flags higher, behind packet?